src/tito/tagger/main.py

```diff
@@ -405,7 +405,7 @@ def _bump_version(self, release=False, zstream=False, force=False):
                                         increase_zstream(match.group(2)),
                                         "\n"
                         ))
-                elif force:
+                elif self._use_version is not None:
```

Comment:
I am ok with removing ForceVersionTagger, that breakage will be clearly visible to anyone that hits it so please feel free to drop that and adjust the places it's imported.

The one issue I'm hoping to avoid is a customer tagger out in the wild, which does not subclass VersionTagger. Suddenly they do a tag --use-version and it no longer does what it did before (because the override to ForceVersionTagger is gone), and just silently runs their normal tagger as if they didn't --use-version. 

I like the goal you've got here to start letting the taggers themselves decide what to do with --use-version I just want to make sure we don't silently alter any behavior on anyone. This is why I'm thinking we should implement a "def supports_use_version(self)" or equivalent, and have it set to true on VersionTagger. We would check if --use-version is set in the CLI code, check that method on that tagger, and throw an error if it's not supported. Then in the scenario above we get a clear error that you can't use --use-version with tagger X and why, and they know they need to adjust their tagger now.